examples/install_in_container.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e7b3ff58-6ab9-436a-a457-1bc12d29b7a8",
+   "metadata": {},
+   "source": [
+    "# Containerized Deployment with Docker"
```

Comment:
this file is not really a notebook, suggest changing it to RST so we can use it in documentation.